server/config.js

```diff
@@ -95,27 +95,212 @@ module.exports = {
     showStackTraces:
       getEnv('ENABLE_STACK_TRACES_ON_ERROR_PAGES', 'false') === 'true',
     useRedisCache: getEnv('ENABLE_REDIS_CACHE', 'true') === 'true',
-    approvedVisitorsFeatureEnabled:
-      getEnv('APPROVED_VISITORS_FEATURE_ENABLED', 'false') === 'true',
-    adjudicationsFeatureEnabled:
-      getEnv('ADJUDICATIONS_FEATURE_ENABLED', 'false') === 'true',
-    adjudicationsFeatureEnabledAt: [
-      'cookhamwood',
-      'erlestoke',
-      'felthama',
-      'felthamb',
-      'garth',
-      'lindholme',
-      'newhall',
-      'ranby',
-      'stokeheath',
-      'styal',
-      'swaleside',
-      'themount',
-      'wayland',
-      'werrington',
-      'wetherby',
-    ],
+  },
+  sites: {
+    berwyn: {
+      enabled: false,
+      features: [],
+    },
+    bullingdon: {
+      enabled: true,
+      features: [
+        'approvedVisitors',
+        'incentives',
+        'money',
+        'timetable',
+        'visits',
+      ],
+    },
+    chelmsford: {
+      enabled: true,
+      features: ['incentives', 'money', 'timetable', 'visits'],
+    },
+    cookhamwood: {
+      enabled: true,
+      features: [
```

Comment:
It may be worth creating a const called something like `ALL_FEATURES` for the prisons that should have access to all features, purely to avoid repetition of this array. Then just define the array for the prisons where there's simply none/some of the features enabled? 😄

Reply:
That makes sense, I'll have a look 👍